tello_driver/CMakeLists.txt

```diff
@@ -99,24 +107,72 @@ target_compile_definitions(tello_emulator
 # Tello joy node
 #=============
 
-add_executable(tello_joy src/tello_joy.cpp)
+add_library(tello_joy_node SHARED
+  src/tello_joy_node.cpp)
+
+target_compile_definitions(tello_joy_node
+  PRIVATE "COMPOSITION_BUILDING_DLL")
 
-ament_target_dependencies(tello_joy
+ament_target_dependencies(tello_joy_node
+  class_loader
   geometry_msgs
   rclcpp
   sensor_msgs
   tello_msgs)
 
+rclcpp_components_register_nodes(tello_joy_node "tello_joy::TelloJoyNode")
+set(node_plugins "${node_plugins}tello_joy::TelloJoyNode;$<TARGET_FILE:tello_joy_node>\n")
+
+#=============
+# Export incantations, see https://github.com/ros2/demos/blob/master/composition/CMakeLists.txt
+#=============
+
+if (NOT WIN32)
+  ament_environment_hooks("${ament_cmake_package_templates_ENVIRONMENT_HOOK_LIBRARY_PATH}")
+endif ()
+
+#=============
```

Comment:
So at this point, we cannot use Clion to debug the code in tello_driver?

Reply:
Ah, right. I'll add a _debug executable.

Reply:
PTAL, the _main executables can be debugged in CLion.